Makefile

```diff
@@ -105,12 +113,6 @@ pyright:
 	xdsl-stubgen
 	pyright $(shell git diff --staged --name-only  -- '*.py')
 
-# run black on all files currently staged
-black:
```

Comment:
Is it worth having a "make ruff" replacement for this or is it just unnecessary?

Reply:
I initially changed this to `make format` but then I thought that was silly since it's the same number of characters and doesn't seem particularly useful.